spec/adapters/datatables_spec.rb

```diff
@@ -14,8 +16,16 @@
   end
 
   describe "initialization Tables" do
-    it "Table class must be Daru::DataTables::DataTable " do
+    it "Table class must be Daru::DataTables::DataTable" do
       expect(Daru::View::Table.new.table).to be_a Daru::DataTables::DataTable
     end
+
+    it "Table class must be Daru::DataTables::DataTable when data objects are not of class Array" do
```

Comment:
Please try to make line length maximum around 80 character. It will be better if you install some extension for it in text editor .